packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -410,10 +459,28 @@ function Menu(props: MenuProps): React.ReactElement {
 			</MenuSection>,
 		);
 	}
-
 	return (
 		<Stack.Item styles={menuStyles}>
 			{menuSections.length === 0 ? <Waiting /> : menuSections}
+			<div
+				style={{
+					minWidth: "250px",
+					display: "grid",
+					gridTemplateRows: "repeat(1fr)",
+					justifyItems: "start",
+				}}
+			>
+				<label style={{ fontSize: "12px" }}>Select theme</label>
+				<Dropdown
+					placeholder="Theme"
+					style={{ minWidth: "150px", fontWeight: "bold" }}
+					onOptionSelect={handleThemeChange}
+				>
+					<Option>Light</Option>
+					<Option>Dark</Option>
+					<Option>High Contrast</Option>
+				</Dropdown>
+			</div>
```

Comment:
I think this would make more sense (and keep the CSS cleaner) in a new menu section + new view `Settings`. What do you think?

Reply:
This is a great idea. Even if the theme is the only setting we have now, it's a good idea to abstract all settings into a new section. Will update.